src/features/percentileCalculator/pages/PercentileCalculatorPage.mock.ts

```diff
@@ -8,26 +8,54 @@ export const mockStations = [
 export const mockPercentilesResponse: PercentilesResponse = {
   stations: {
     [mockStations[0].code]: {
-      ffmc: 94.95859222412109,
-      isi: 15.589389991760253,
-      bui: 157.5261016845703,
+      ffmc: 94.2121,
+      isi: 15.4124,
+      bui: 157.2321,
       season: { start_month: 5, start_day: 1, end_month: 8, end_day: 31 },
       years: [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019],
       station: { ...mockStations[0] }
     },
     [mockStations[1].code]: {
-      ffmc: 90.95859222412109,
-      isi: 10.589389991760253,
-      bui: 159.5261016845703,
+      ffmc: null,
+      isi: null,
+      bui: null,
       season: { start_month: 5, start_day: 1, end_month: 8, end_day: 31 },
       years: [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019],
       station: { ...mockStations[1] }
     }
   },
   mean_values: {
-    ffmc: 93.95859222412109,
-    isi: 12.589389991760253,
-    bui: 158.5261016845703
+    ffmc: 94.2121,
+    isi: 15.4124,
+    bui: 157.2321
+  },
+  year_range: { start: 2010, end: 2019 },
+  percentile: 90
+}
+
+export const mockNullPercentilesResponse: PercentilesResponse = {
+  stations: {
+    [mockStations[0].code]: {
+      ffmc: null,
+      isi: null,
+      bui: null,
+      season: { start_month: 5, start_day: 1, end_month: 8, end_day: 31 },
```

Comment:
heads up because this will probably cause issues when my PR gets merged, but `season` is no longer used because it's not needed anymore. Instead the station class has `core_season` within it

Reply:
Right... I need to make more changes then!